### PR TITLE
Objects Automated Test - LPS-161897 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-161897

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 1]
```diff
@@ -1378,6 +1368,8 @@ definition {
 	@description = "LPS-147792 - Verify it is possible to see the column label as the alias on the view builder column list"
 	@priority = "5"
 	test CanSeeRenamedColumnOnViewBuilder {
+		property portal.acceptance = "true";
+		
 		ObjectAdmin.addObjectViaAPI(
 			labelName = "Custom Object 152068",
 			objectName = "CustomObject152068",
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:1375]: This shouldn't be part of the commit `LPS-152066 Remove duplicate empty stub ` since you're adding a test to Acceptance. Please do this in a separate commit.

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 2]
```diff
@@ -2963,6 +2963,63 @@ definition {
...
+		}
+
+		task ("Assert that it's not possible to update Account Restriction") {
+			WaitForSPARefresh();
+
+			CreateObject.assertAccountRestrictedFieldDisabled(valueName = "RelationshipAccount");
+
+			CreateObject.viewToogleAccountRestrictionDisabled();
+		}
+	}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:3021]: I'm a little confused here, are you asserting that it's not possible to disable the account restriction here?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 3]
```diff
@@ -1369,7 +1369,7 @@ definition {
 	@priority = "5"
 	test CanSeeRenamedColumnOnViewBuilder {
 		property portal.acceptance = "true";
-		
+
 		ObjectAdmin.addObjectViaAPI(
 			labelName = "Custom Object 152068",
 			objectName = "CustomObject152068",
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:1375]: If you need to make an SF change, please do it in the commit that caused it.

**Review by timpak — CHANGES_REQUESTED** [review]
Hey @andreefarias, please see my question for the test and make the following change:
- Please add the property `property portal.acceptance = "true";` to `CanSeeRenamedColumnOnViewBuilder ` in a separate commit. Make sure your SF change is included in this commit. You can commit this under the ticket that added the test.
